torch/_dynamo/utils.py

```diff
@@ -278,12 +278,12 @@ def setup_compile_debug():
     compile_debug = os.environ.get("TORCH_COMPILE_DEBUG", "0") == "1"
 
     if compile_debug:
-        torch._logging.set_logs(
-            dynamo=logging.DEBUG,
-            aot=logging.DEBUG,
-            inductor=logging.DEBUG,
-            output_code=True,  # this is off by default
-        )
+        # torch._logging.set_logs(
+        #     dynamo=logging.DEBUG,
+        #     aot=logging.DEBUG,
+        #     inductor=logging.DEBUG,
+        #     output_code=True,  # this is off by default
+        # )
```

Comment:
intended?

Reply:
Yeah this stuff completely mucks up the `TORCH_COMPILE_DEBUG` output.